application/src/main/java/org/togetherjava/tjbot/commands/moderation/attachment/package-info.java

```diff
@@ -1,5 +1,7 @@
 /**
- * This packages offers all the functionality for checking attachment safety.
+ * This packages offers all the functionality for checking attachment safety see
+ * {@link org.togetherjava.tjbot.commands.moderation.attachment.BlacklistedAttachmentListener} as
```

Comment:
```suggestion
 * {@link org.togetherjava.tjbot.commands.moderation.attachment.BlacklistedAttachmentListener BlacklistedAttachmentListener} as
```
Otherwise it shows up as the FQN in the docs

Reply:
fixed